suite-native/module-send-receive/src/components/SendReceiveBottomSheet.tsx

```diff
@@ -77,13 +86,5 @@ export const SendReceiveBottomSheet = ({
         },
     };
 
-    return (
-        <BottomSheet
-            isVisible={isVisible}
-            onVisibilityChange={handleClose}
-            title={sendReceiveContent[contentType].title}
-        >
-            {sendReceiveContent[contentType].component}
-        </BottomSheet>
-    );
+    return <Screen header={<ScreenHeader />}>{sendReceiveContent[contentType].component}</Screen>;
```

Comment:
is `sendReceiveContent` in `useMemo`?

Reply:
Not at the moment. But I guess it should be to avoid re-creation of the whole object since it will never really change unless props do. Good point

Reply:
[1930549](https://github.com/trezor/trezor-suite/pull/7060/commits/193054970ef9072baacd801abce08b08fb7b5e09)